mixer/pkg/server/server.go

```diff
@@ -26,6 +26,8 @@ import (
 	ot "github.com/opentracing/opentracing-go"
 	"google.golang.org/grpc"
 
+	"os"
```

Comment:
this should have caused linter issues, no?

Reply:
Linter is completely broken...